### ISSUE TITLE
image/jpeg: add a jpeg option to disable chroma subsampling

### ISSUE THREAD

**Issue by urandom** [golang/go#13614]
It seems that golang's jpeg encoder only supports downsampling. It would be nice to be able to fine-tune what kind of subsampling is used when encoding jpegs, or alternatively disable downsampling when using a large quality value.

This is a continuation of:
https://groups.google.com/forum/#!topic/golang-nuts/VJY87VQpgws

**Comment by nigeltao** [comment]
Yeah, it's a valid bug, but I don't know when I'll have time to fix it.
